src/api/app/models/user_ldap_strategy.rb

```diff
@@ -339,12 +339,10 @@ def self.initialize_ldap_con(user_name, password)
     return unless defined?(CONFIG['ldap_servers'])
     require 'ldap'
     ldap_servers = CONFIG['ldap_servers'].split(":")
-    count = 0
 
     max_ldap_attempts = CONFIG.has_key?('ldap_max_attempts') ? CONFIG['ldap_max_attempts'] : 10
 
-    while count < max_ldap_attempts
-      count += 1
+    (max_ldap_attempts || 10).times do
```

Comment:
The thing is that I have another bunch of commits, but I split them. And obviously things got mixed up here.